requirements.txt

```diff
@@ -1,36 +1,37 @@
 -i https://pypi.org/simple
 -e .
-attrs==19.3.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+attrs==20.2.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
 authlib==0.14.1
 blinker==1.4
 certifi==2020.6.20
-cffi==1.14.0
+cffi==1.14.2
 chardet==3.0.4
 click==7.1.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
-cryptography==2.9.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
+cryptography==3.1; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
```

Comment:
I am a little concerned about these (meaning, `cryptography` and `attrs` (above)) being updated to new major versions. Do you know why that's happening? Is it because we have `>=` in our Pipfile, and it should really be `>=, <` or something like that?

Reply:
We don't directly use these dependencies, they're sub-dependencies of our actual dependencies. It looks like the bump in the `cryptography` version number is due to the addition of `pyjwt`, as it uses that package. I don't think we should really be trying to control the package versions of our sub-dependencies, so I'm not worried about this.